app/decorators/module_overview_decorator.rb

```diff
@@ -17,7 +17,7 @@ def topics_by_submodule
         num,                                  # submodule digit
         items.first.model.heading,            # submodule intro heading
         previous_submodule_heading_text(num), # previous submodule heading
-        status(items),                        # symbol  - all items in the submodule
```

Comment:
is this the same place we discussed before where we could also return different items? We need the first item for line 18. Should we split them in two?

Reply:
Perhaps we should have 
```ruby 
intro = items.first
content = items[1..]
```
at the start of the block?

Reply:
Yeah that makes sense. Have made the change